src/docs/arch-block.dox

```diff
@@ -15,6 +15,35 @@ re-written, they are written to new locations in the file. The size of a block
 is a multiple of the allocation size which is set during creation of the
 associated WiredTiger data file see: WT_SESSION::create.
 
+@plantuml_start{block.png }
+@startuml{block.png}
+
+title Block Layout
```

Comment:
If you write this as: Diagram: Block Layout it becomes easy to search for all diagrams in the manual.

Reply:
Are you referring to the search bar in the docs webpage or just a way to search the docs source code?

This corresponds to the text at the top of the generated diagram. Is this what you're suggesting?

<img width="727" alt="Screenshot 2024-07-01 at 10 26 52 AM" src="https://github.com/wiredtiger/wiredtiger/assets/41533874/a4d1f4e5-6776-48d2-ad8e-7497c53fb8a8">

Reply:
* Both. If we have a standard way of titling diagrams it makes searching for all of them easy wherever that search is done.
* Yes, that is what I meant. Then searching for Diagram: will find all diagrams.